CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * Added the **convert** command to convert entities between XSOAR versions.
 * Added the *--deprecate* flag in **format** command to deprecate integrations, scripts, and playbooks.
 * Fixed an issue where ignoring errors did not work when running the **validate** command on specific files (-i).
+* Added a validation to ensure integrations are not skipped and at least one test playbook is not skipped for each integration or script.
```

Comment:
move this changelog to the correct header please.

Reply:
what do you mean?